vet-service/src/main/java/com/petclinic/vet/dataaccesslayer/Photo.java

```diff
@@ -0,0 +1,20 @@
+//package com.petclinic.vet.dataaccesslayer;
```

Comment:
Is this class necessary?

Reply:
For my second story (694) which is connected to this user storie will need this package for photos implementation in Backend.